packages/app-admin-ui/client/components/CreateItemModal.js

```diff
@@ -244,10 +271,36 @@ const CreateItemModal = ({ prefillData = {}, onClose, onCreate, viewOnSave }) =>
               ));
             }}
           </Render>
+          <ConfirmModal
+            isOpen={isConfirmOpen}
+            onYes={() => {
```

Comment:
I'd call this `onConfirm` and `onCancel`

Reply:
I'd also move these inline functions to callbacks too.

```
 <ConfirmModal
  isOpen={isConfirmOpen}
  onConfirm={onConfirm}
  onCancel={onCancel}
/>
```

Reply:
Yeah that was exactly my comment :)

Reply:
What's the value in moving an inline function unless it's used more than once or is huge? 

My preference is usually to keep the code as small as possible until it hinders readability.